source/core/src/main/com/csse3200/game/inventory/items/lootbox/configs/MediumGameLootTable.java

```diff
@@ -0,0 +1,22 @@
+package com.csse3200.game.inventory.items.lootbox.configs;
+
+import com.csse3200.game.inventory.items.food.Foods;
+import com.csse3200.game.inventory.items.potions.AttackPotion;
+import com.csse3200.game.inventory.items.potions.DefensePotion;
+import com.csse3200.game.inventory.items.potions.HealingPotion;
+import com.csse3200.game.inventory.items.potions.SpeedPotion;
+
+public class MediumGameLootTable extends BaseLootTable {
+    public MediumGameLootTable() {
+
+        addItem(HealingPotion.class, 100.0, new Class<?>[]{int.class}, new Object[]{1});
+        addItem(Foods.Apple.class, 10.0, new Class<?>[]{int.class}, new Object[]{1});
```

Comment:
As before with config file. We can probably create a big config file for all items and a universal constructor - that way it can be used for this and for saving/loading items from a config file.